frontend/src/components/Header.tsx

```diff
@@ -53,30 +53,32 @@ const HeaderNoCtx: React.FC<ContextType> = (props) => {
     apiGet
   } = useAuthContext();
   const [navOpen, setNavOpen] = useState(false);
-  const [organizations, setOrganizations] = useState<Organization[]>([]);
-
-  const fetchOrganizations = useCallback(async () => {
-    try {
-      let rows = await apiGet<Organization[]>('/organizations/');
-      setOrganizations(rows);
-    } catch (e) {
-      console.error(e);
-    }
-  }, [apiGet, setOrganizations]);
+  const [organizations, setOrganizations] = useState<
+    (Organization | OrganizationTag)[]
+  >([]);
 
   let userLevel = 0;
   if (user && user.isRegistered) {
     if (user.userType === 'standard') {
-      if (currentOrganization?.userIsAdmin) {
-        userLevel = ORG_ADMIN;
```

Comment:
If we're removing this anyway, should we just rename `ORG_USER` to `STANDARD_USER` and remove `ORG_ADMIN`?